program/src/processor.rs

```diff
@@ -649,17 +649,17 @@ pub fn check_address_and_get_balance(
 
 /// Recover any inactive balance from the validator's stake/unstake accounts to
 /// the reserve account.
-/// Updates the validator's balance.
+/// Updates the validator's balance and distribute rewards.
 /// This function is permissionless and can be called by anyone.
 pub fn process_withdraw_inactive_stake(
     program_id: &Pubkey,
     raw_accounts: &[AccountInfo],
 ) -> ProgramResult {
-    let accounts = WithdrawInactiveStakeInfo::try_from_slice(raw_accounts)?;
+    let accounts = WithdrawInactiveStakeInfoV2::try_from_slice(raw_accounts)?;
```

Comment:
Now that rewards show up primarily as increased _active_ balance of the stake accounts, this instruction does more than just withdrawing inactive stake, it is _the_ primary mechanism through which fees get distributed. I think it would be clearer for readers if we reflect that in the name, maybe `UpdateStakeAccountBalance` or something like this.

Reply:
Yes, `UpdateStakeAccountBalance ` sounds good. It distributes fees and it also withdraws inactive stake, but the main thing it does is updating a state variable `stake_accounts_balance`. Fixed